src/lib/logstore/log_store_service.cpp

```diff
@@ -97,8 +97,6 @@ std::shared_ptr< VirtualDev > LogStoreService::open_vdev(const vdev_info& vinfo,
 void LogStoreService::start(bool format) {
     // hs()->status_mgr()->register_status_cb("LogStore", bind_this(LogStoreService::get_status, 1));
 
-    delete_unopened_logdevs();
```

Comment:
We let raft repl service to call this after it has loaded all logdev's. Otherwise there is a concurrency issue during restart.